sampleAppKotlin/src/main/AndroidManifest.xml

```diff
@@ -21,6 +21,31 @@
             </intent-filter>
         </activity>
         <activity android:name=".ProductActivity" />
+
+        <!--
+            SNS authentication setup:
+            Include an intent-filter and set the android:scheme on the VitrusizeAuthActivity that will be responsible for handling the deep link back into the app
+        -->
+        <activity
+            android:name="com.virtusize.android.auth.views.VitrusizeAuthActivity"
+            android:exported="true"
+            android:launchMode="singleTask">
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <!--
+                    The URL host has to be sns-auth and the URL scheme must begin with your app's package ID (com.company.app) and end with .virtusize
+                    Note: The scheme you define must use all lowercase letters.
+                -->
+                <data
+                    android:host="sns-auth"
+                    android:scheme="com.virtusize.sampleappkotlin.virtusize" />
+            </intent-filter>
+        </activity>
```

Comment:
Clients have to include this into their AndroidMainfest.xml to handle deep links from the SNS (Facebook) auth page loaded in a Custom Custom Tab back into the app